dags/surveymonkey_data_import_pipeline.py

```diff
@@ -117,6 +119,39 @@ def surveymonkey_survey_questions_etl(**kwargs):
         )
 
 
+def surveymonkey_survey_answers_etl(**kwargs):
+    data_config = data_config_from_xcom(kwargs)
+    survey_id_list_from_df = get_distinct_values_from_bq(
+        project_name=data_config.project_name,
+        dataset_name=data_config.dataset_name,
+        table_name_source=data_config.survey_list_table_name,
+        column_name=data_config.survey_id_column_name
+    ).values.tolist()
+
+    for survey_id_list in survey_id_list_from_df:
+        survey_id = str(survey_id_list[1])
+        LOGGER.info("answers for survey_id: %s", survey_id)
+        list_of_answers_of_one_survey = get_survey_answer_details(
+            access_token=get_surveymonkey_access_token(),
+            survey_id=survey_id
+        )
+        len_of_answer_list = len(list_of_answers_of_one_survey)
+        LOGGER.info("length of answers is %s", len_of_answer_list)
+        if len_of_answer_list > 0:
```

Comment:
minor: I believe if you inline `len` to `if len(list_of_answers_of_one_survey) > 0:` then `pylint` will complain.

That is because the more `pythonic` way is to just use `if list_of_answers_of_one_survey:`

Reply:
Actually I used a variable because I put the length in logging too, so not to call it two times, if it make sense?

Reply:
I have changed it as I don't need to call one more time :)